news-bundle/src/Controller/Page/NewsFeedController.php

```diff
@@ -42,7 +42,7 @@ class NewsFeedController extends AbstractController implements DynamicRouteInter
         'rss' => '.xml',
     ];
 
-    public function __construct(private readonly ContaoContext $contaoContext, private readonly Specification $specification)
+    public function __construct(private readonly ContaoContext $contaoContext, private readonly Specification $specification, private readonly string $charset)
```

Comment:
We do not support any charset but `UTF-8` in Contao 5, so this does not have to be dynamically IMHO. @contao/developers WDYT?

Reply:
I think it’s still goot practise to use `%kernel.charset%` where possible.